roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -73,6 +73,11 @@ servingInfo:
   bindAddress: 0.0.0.0:10250
   bindNetwork: tcp4
   clientCA: client-ca.crt
+{% if openshift_use_calico | default(False) | bool %}
```

Comment:
While OpenShift SDN might not need it for this particular case, there are a handful of edge cases that kube-proxy uses the `cluster-cidr` argument for, so it's probably best to set it regardless of network plugin.

Reply:
That's fair. I originally kept it specific to Calico since I wasn't sure what the effect would be with other network plugins. I can make those changes.